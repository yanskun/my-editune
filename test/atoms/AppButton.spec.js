import { shallowMount } from "@vue/test-utils"
import Component from "~/components/atoms/AppButton"

describe("AppButton", () => {
  test("snapshot", () => {
    const wrapper = shallowMount(Component, {
      propsData: {
        type: "primary"
      }
    })
    expect(wrapper.element).toMatchSnapshot()
  })
})
