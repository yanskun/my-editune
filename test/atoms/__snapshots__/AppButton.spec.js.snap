// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppButton snapshot 1`] = `
<el-button-stub
  icon=""
  nativetype="button"
  type="primary"
/>
`;
